documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,405 @@
+
+.The `VirtualMachineClusterFlavor` CRDs replace {ocp-name} templates
+
+The Virtual Machine Flavor API replaces {ocp-name} templates. {project-short} currently falls back to using {ocp-name} templates when a relevant preference is not available.
+
+.Full support for migration from OVA
+
+{project-short} {project-version} supports migration from OVA.
+
+.{ocp-name} Console `Create` and `Edit` pages replace Forms and Wizards
+
+Forms and Wizards that use `@migtools/lib-ui` do not function as expected when running inside the plugin. The migtool library was built as a standalone application. The `Create` and `Edit` pages in the {ocp-name} Console replace the Forms and Wizards. link:https://issues.redhat.com/browse/MTV-615[(MTV-615)]
+
+.Scan OVA files in the root folder of an NFS share
+
+Migration scans the files directly to the NFS share folder, not to a separate folder. link:https://issues.redhat.com/browse/MTV-680[(MTV-680)]
+
+
+[id="new-features-and-enhancements-26_{context}"]
+== New features and enhancements
+
+This release has the following features and improvements:
+
+.Replace templates with flavors
+
+KubeVirt API introduces `VirtualMachineClusterFlavor` and `VirtualMachineFlavor` Containerized Data Importers(CDIs). Flavors are meant to replace the Templates, which are only available on {ocp-name}. link:https://issues.redhat.com/browse/MTV-206[(MTV-206,] link:https://issues.redhat.com/browse/MTV-919[MTV-919)]
+
+.Integrate volume populators in remote migrations
+
+// Arik, please check and explain - `el9` > "Red Hat Enterprise Linux 9"?
+Integrate volume populator for `oVirt` and OpenStack, together with `virt-v2v` on `el9` into a CDI to migrate remote {ocp-name} clusters. link:https://issues.redhat.com/browse/MTV-721[(MTV-721)]
+
+.VM posted with `Running` state
+
+{project-short} shows each VM's migration `Running` state during the migration. link:https://issues.redhat.com/browse/MTV-794[(MTV-794)]
+
+.Review and configure migrated VMs
+
+The {project-short} web console shows the name and migration status of each VM. link:https://issues.redhat.com/browse/MTV-843[(MTV-843)]
+
+.Default virt storage class
+// change to virtual?
+
+For requests that do not specify an explicit storage class, the `is-default-virt-class` replaces the Kubernetes default storage class. link:https://issues.redhat.com/browse/MTV-855[(MTV-855)]
+
+.Validate permissions on source providers
+
+When migrating a VM with an Red Hat Enterprise Virtualization (RHEV) user with restricted permissions, {project-short} validates the user permissions and the migration pod reports where the process fails. link:https://issues.redhat.com/browse/MTV-273[(MTV-273)]
+
+.Validation for RHEL 6
+
+Red Hat Enterprise Linux (RHEL) 9 does not support CNV to provide RHEL 6 Templates. This restriction is no longer relevant because `VirtualMachineClusterFlavor` CRDs have replaced the templates. link:https://issues.redhat.com/browse/MTV-413[(MTV413)]
```

Comment:
```suggestion
Red Hat Enterprise Linux (RHEL) 9 does not support RHEL 6 as a guest operating system and therefore RHEL 6 is not supported as a guest operating system in OpenShift Virtualization. A validation of RHEL 6 guest operating system was added. link:https://issues.redhat.com/browse/MTV-413[(MTV413)]
```
it's not really a feature.. maybe a technical change?

Reply:
changed